airmozilla/main/templates/main/home.html

```diff
@@ -11,7 +11,17 @@
 </style>
 {% endblock %}
 
-
+{% block extra_head %}
+{{ super() }}
+<meta name="keywords" content="{{ ','.join(tags) }}">
+<meta property="og:title" content="Home">
```

Comment:
Wow... I totally confused that part. I was using the tab title instead. I now got it, thanks.